specification/compute/resource-manager/Microsoft.Compute/RecommenderRP/main.tsp

```diff
@@ -0,0 +1,44 @@
+/**
+ * PLEASE DO NOT REMOVE - USED FOR CONVERTER METRICS
+ * Generated by package: @autorest/openapi-to-typespec
+ * Parameters used:
+ *   isFullCompatible: true
+ *   guessResourceKey: false
+ * Version: 0.10.14
+ * Date: 2025-07-18T08:45:04.778Z
+ */
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "./models.tsp";
```

Comment:
Nit: this is already imported through ComputeDiagnosticBase.tsp

Reply:
Removed